source/landing-zone.txt

```diff
@@ -19,193 +19,165 @@ Landing Zone Design
+   * - Compliance
+     - Consider how your deployment's data residency determines data sovereignty
+       and therefore which laws apply to your data. Identify and account for any specific legal and regulatory
+       requirements not clearly articulated within other categories of requirements. 
+
+       Your data residency depends on which regions and geographies you choose to deploy to in your deployment paradigm, 
+       and whether you choose to partition your data between geographies.  
+
+       To get recommendations and learn more about this topic, see the following pages:
+
+       - :ref:`arch-center-compliance`
+       - :ref:`arch-center-paradigms`
+   * - Disaster Recovery
+     - Define and record a disaster recovery plan that meets the following criteria: 
+     
+       - Defines an optimal |rpo| and |rpo| for your organization
+       - Defines a backup snapshot schedule and requirements for snapshot retention and mutli-region distribution 
+       - Identifies recovery methods such as snapshot restores, queryable backups, document versioning, region shifting, 
+         or provider pivot
+       - Defines disaster recovery procedures for possible disaster scenarios such as zonal, regional, or cloud-provider 
+         outages, resource failures, data corruption events, and more. 
 
        To get recommendations and learn more about this topic, see
-       :ref:`arch-center-auditing-logging`.
-   * - Data Residency Requirements
-     - Identify data residency requirements, such as data sovereignty
-       requirements, or application data partitioning rules.
-   * - Network Topology Requirements
-     - Identify network topology requirements, including Cloud Provider
-       regions, private connectivity options, and application
-       deployment topology.
+       the following pages: 
+       
+       - :ref:`arch-center-dr`
+       - :ref:`arch-center-backups`
+   * - High Availability
+     - Set standards for high availability that ensure system operation during planned or 
+       unplanned outages. These requirements will partially determine the number of nodes, 
+       regions, and cloud providers in your deployment paradigm. 
 
        To get recommendations and learn more about this topic, see
-       :ref:`arch-center-network-security`.
-   * - High Availability Requirements
-     - Identify high availability requirements such as cluster topology
-       with node count and priority per region, global write
-       requirements, and partitioning tolerances.
+       the following pages: 
+
+       - :ref:`arch-center-high-availability`
+       - :ref:`arch-center-paradigms`
+   * - Billing
+     - Identify any specific requirements for billing, such as
+       integrations with FinOps tools for reporting and charge-back.
+       You can build these requirements into the automation and
+       provisioning process for {+service+} {+clusters+} to facilitate
```

Comment:
nit pick: we no longer need to use the {+clusters+} source constant. You can replace that with `clusters` throughout your copy. We've moved away from trying to disambiguate dedicated (M10+) Atlas clusters from serverless, so any Atlas deployment can now be called a cluster. We added in the source constant because this had changed a few times while serverless was in development.